### PR TITLE
New product defs for silt,sand, clay content, bedrock depth and bulk …

### PR DESCRIPTION
**Pull request by SeffatChowdhury**
…density

### CODE REVIEW

**Review by fangfy — COMMENTED** [review]
@SeffatChowdhury I updated the units and added back-transformatoin (hopefully in a correct format and will work). Please confirm and also confirm nodata value.

**Comment by SeffatChowdhury** [comment]
@fangfy the no data value seems to be NaN

File: products/isda_soil_bedrock_depth.odc-product.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  crs: EPSG:3857
+  resolution:
+    x: 30
+    y: -30
+
+measurements:
+  - name: mean_0_200
+    aliases: [MEAN_0_200]
+    dtype: float32
+    nodata: 0.0
```
Comment (fangfy) [inline products/isda_soil_bedrock_depth.odc-product.yaml:22]: this and other nodata values should be nan?

**Review by fangfy — APPROVED** [review]

**Comment by SeffatChowdhury** [comment]
Thank you @fangfy

**Pull request merged**